diff --git i/packages/opencode/script/build.ts w/packages/opencode/script/build.ts
index 97631d907..917891a5c 100755
--- i/packages/opencode/script/build.ts
+++ w/packages/opencode/script/build.ts
@@ -31,27 +31,27 @@ const allTargets: {
     os: "linux",
     arch: "x64",
   },
-  {
-    os: "linux",
-    arch: "x64",
-    avx2: false,
-  },
-  {
-    os: "linux",
-    arch: "arm64",
-    abi: "musl",
-  },
-  {
-    os: "linux",
-    arch: "x64",
-    abi: "musl",
-  },
-  {
-    os: "linux",
-    arch: "x64",
-    abi: "musl",
-    avx2: false,
-  },
+  // {
+  //   os: "linux",
+  //   arch: "x64",
+  //   avx2: false,
+  // },
+  // {
+  //   os: "linux",
+  //   arch: "arm64",
+  //   abi: "musl",
+  // },
+  // {
+  //   os: "linux",
+  //   arch: "x64",
+  //   abi: "musl",
+  // },
+  // {
+  //   os: "linux",
+  //   arch: "x64",
+  //   abi: "musl",
+  //   avx2: false,
+  // },
   {
     os: "darwin",
     arch: "arm64",
@@ -60,20 +60,20 @@ const allTargets: {
     os: "darwin",
     arch: "x64",
   },
-  {
-    os: "darwin",
-    arch: "x64",
-    avx2: false,
-  },
-  {
-    os: "windows",
-    arch: "x64",
-  },
-  {
-    os: "windows",
-    arch: "x64",
-    avx2: false,
-  },
+  // {
+  //   os: "darwin",
+  //   arch: "x64",
+  //   avx2: false,
+  // },
+  // {
+  //   os: "windows",
+  //   arch: "x64",
+  // },
+  // {
+  //   os: "windows",
+  //   arch: "x64",
+  //   avx2: false,
+  // },
 ]
 
 const targets = singleFlag
